app/assets/javascripts/discourse/app/helpers/value-entered.js

```diff
@@ -0,0 +1,11 @@
+import { registerUnbound } from "discourse-common/lib/helpers";
+
+registerUnbound("value-entered", function (value) {
+  if (value === undefined || value === null) {
```

Comment:
what haopens if value is false?

Reply:
Im not sure the value passed here would ever be `false` as it is either filled in text, or nothing at all, which I encountered as `null` or `undefined`. When nothing is passed in, `""` is returned in order to add no class to the element.

Reply:
you just do if(!value) then